### PR TITLE
(PA-1568) Use stomp v1.4.4 to drop patch, fix Ruby warning

### PR DESCRIPTION
**Pull request by MikaelSmith**
With Ruby 2.4, stomp generates a warning: `constant ::TRUE is deprecated`. Update to stomp v1.4.4 to fix it. That also allows us to drop patches that have been incorporated into the project.

### CODE REVIEW

**Comment by MikaelSmith** [comment]
Targeting this at 5.3.1.

**Comment by joshcooper** [comment]
I thought we intentionally didn't bump to stomp 1.4 because we were concerned about other incompatible changes (in addition to the MITM fix)? @shrug may remember?

**Comment by MikaelSmith** [comment]
There were things that were broken. They’ve been fixed in 1.4.4. I’ve done some testing that this now works.

**Comment by MikaelSmith** [comment]
I’ll double check that both patches are in 1.4.4 now. Pretty sure they are.

File: resources/patches/ruby-stomp/connection_loss_reconnect_fix.patch [hunk 1]
```diff
@@ -1,125 +0,0 @@
-From ed5da4d1d857c427bf995e50c9baa25befaa3175 Mon Sep 17 00:00:00 2001
```
Comment (MikaelSmith) [inline resources/patches/ruby-stomp/connection_loss_reconnect_fix.patch]: This patch is present at https://github.com/stompgem/stomp/commit/dea38701d93dd15dac59b3b5dbabb3df2df9dde0.

File: resources/patches/ruby-stomp/verify_client_certs.patch [hunk 2]
```diff
@@ -1,69 +0,0 @@
-From d18bb48b93d0af2ebdb936811fb980706dfcecf8 Mon Sep 17 00:00:00 2001
```
Comment (MikaelSmith) [inline resources/patches/ruby-stomp/verify_client_certs.patch]: Present at https://github.com/stompgem/stomp/commit/4b4d09009b6011d46f948600cdb935980243fbe5 and  https://github.com/stompgem/stomp/commit/3b9fb734a6508de9f7c9769894cc3a8a0ad8b252.

**Comment by MikaelSmith** [comment]
Any additional concerns?

**Comment by MikaelSmith** [comment]
Holding off merging until 5.3.1_release is branched.

**Pull request merged**